### PR TITLE
Create unit tests

### PR DESCRIPTION
**Pull request by trace-kadenyi**
Use jest to create units tests for the components (using snapshots), the pages, and the logic files:
- Calculator.test.js
- Navbar.test.js
- calculate.test.js
- operate.test.js
- Home.test.js
- Quote.test.js

### CODE REVIEW

File: src/Logic/__test__/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  });
+  test('it subtracts numbers', () => {
+    expect(operate(2, 2, '-')).toBe('0');
+  });
+  test('it multiplies numbers', () => {
+    expect(operate(2, 2, 'x')).toBe('4');
+  });
+  test('it divides numbers', () => {
+    expect(operate(2, 2, '÷')).toBe('1');
+  });
```
Comment (tufoinnkuo10) [inline src/Logic/__test__/operate.test.js:15]: -[ ] great job with the operations.. kindly add an operation test for modulo`(%)`

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @TracK92 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ React testing library and jest used

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/Logic/__test__/operate.test.js (hunk 1 above)
Comment (trace-kadenyi) [reply src/Logic/__test__/operate.test.js:15]: [tufoinnkuo10](https://github.com/tufoinnkuo10), thanks for your review. I'll make the changes.

File: src/Logic/__test__/operate.test.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
```
Comment (Nemwel-Boniface) [inline src/Logic/__test__/operate.test.js:15]: - [ ] [OPTIONAL] - Seeing tests in projects makes everyone happy :smiley: . What if you now stretch it to test scenarios such as where there are negative values in the operations?

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @TracK92  and @dodoburner              :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the changes requested by the previous reviewer good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by trace-kadenyi** [comment]
@Nemwel-Boniface, thanks. We'll work on it.

**Pull request merged**